### DISCUSSION TITLE
Session key decryption failed.

### DISCUSSION THREAD

**Discussion by awasank** [openpgpjs/openpgpjs#1391] [Q&A]
![Capture](https://user-images.githubusercontent.com/82606961/127362189-caa743fc-5bcf-4c29-bb70-8b84081c78ae.PNG)

I know this problem has been addressed in the past with a possible solution to add a flag. #1126
**openpgp.config.allow_insecure_decryption_with_signing_keys = true;**
- OpenPGP.js version: 5.0.0-5
- Affected platform (Browser or Node.js version): Node.js

I am able to encrypt and decrypt a file even without the flag. I have shared the public key with one my client. But I get this error while decrypting the encrypted file they send. I have tried putting this flag but it doesn't seem to work. I know the file they sent is correct because I use my private key to decrypt it on https://8gwifi.org/pgpencdec.jsp and it works just fine. Not sure if I am missing something or is this related to the issue previously discussed. Any suggestion is really appreciated.

**Comment by larabr** [comment]
Hi @awasank, in v5 we changed the casing of `config` options. Try setting `config.allowInsecureDecryptionWithSigningKeys` (related: https://github.com/openpgpjs/openpgpjs/discussions/1352#discussioncomment-931652).
